### ISSUE TITLE
@moniker ruins @mail formatting

### ISSUE THREAD

**Issue by brazilofmux** [brazilofmux/tinymux#201]
Original [issue 198](https://code.google.com/p/tinymux/issues/detail?id=198) created by brazilofmux on 2006-10-26T16:23:56.000Z:

Receive mail from someone with a @moniker set.  You'll discover that the Subject line no longer 
lines up with the other subject lines, as the alignment is done including ANSI, rather than excluding 
it.

<b>What is the expected output? What do you see instead?</b>
The expected output is that the Subject field should still line up with other Subject fields, but it 
does not.  This likely affects more than @mail, but @mail is the place where the aesthetic quirk 
becomes annoying.

In short, spacing alignments should be done on the ANSI-less string, rather than the ANSI-fied 
string version. :)
